### ISSUE TITLE
Nothing but index page works with current versions

### ISSUE THREAD

**Issue by kylebolstad** [ktsn/vue-cli-plugin-auto-routing#7]
I just set up a project from scratch, and all that seems to work is the `index` page. The `about` page just shows the root `index` page as well. I ran the sample code from issue [#1](https://github.com/ktsn/vue-cli-plugin-auto-routing/issues/1#issuecomment-407132951) just fine, but when I updated to the following versions it, too, broke.

**dependencies**

`"vue": "^2.5.21",
    "vue-router": "^3.0.1",
    "vue-router-layout": "^0.1.2"`

**devDependencies** 

`"vue-auto-routing": "^0.3.0",
    "vue-cli-plugin-auto-routing": "^0.2.1",
    "vue-template-compiler": "^2.5.21"`

**Comment by ktsn** [comment]
Thanks for filing the issue. Please provide a self-contained reproduction to confirm.

**Comment by ndarilek** [comment]
Ugh, I just hit this one too. Pretty vanilla app. I'm not sure what in this fairly minimal app may have caused it, but I'll try packaging it tomorrow and will report back. Spent way too much time on this today, but wanted to confirm that routing isn't working for me either.

**Comment by ndarilek** [comment]
To get this working, I had to create my router like so:

```
const router = new Router({
  mode: "history",
...
```

That is, I had to explicitly set the mode. I created a basic app and everything worked fine. Somewhere between creating that basic app and arriving where I am now, the default mode was changed, and all my routes actually had the hash *after* the path for some odd reason (I.e. /login#/). Not sure what's up with that, but explicitly specifying the mode got it working.
